package-lock.json

```diff
@@ -1,2034 +1,8 @@
 {
   "name": "content",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
@ilaner please revert changes to this file.

Reply:
fixed